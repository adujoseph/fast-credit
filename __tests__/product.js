import React from 'react';
import renderer from 'react-test-renderer';
import Intro from '../src/screen/productDetails/ProductDetails';

test('product renders correctly', () => {
  const tree = renderer.create(<Intro />).toJSON();
  expect(tree).toMatchSnapshot();
});
