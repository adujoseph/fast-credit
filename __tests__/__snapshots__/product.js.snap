// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`product renders correctly 1`] = `
<View>
  <Text>
    Notification Screen
  </Text>
</View>
`;
